### PR TITLE
Fix code style of crypto code

### PR DESCRIPTION
**Pull request by mtausig**
### Contribution description

I fixed the code style of all crypto code files.

Uncrustify was run on all relevant source, header and test files.

No functional changes were made.

### Testing procedure

The tests in *sys_crypto* still work.

### Issues/PRs references

This was requested by @aabadie in PR #12362.

### CODE REVIEW

File: sys/crypto/chacha20poly1305.c [hunk 1]
```diff
@@ -83,7 +84,7 @@ void _keystream(chacha20poly1305_ctx_t *ctx, const uint8_t *key,
 
     /* perform rounds */
     for (unsigned i = 0; i < 80; ++i) {
-        uint32_t *a = &ctx->state[((i                    ) & 3)          ];
+        uint32_t *a = &ctx->state[((i) & 3)          ];
```
Comment (aabadie) [inline sys/crypto/chacha20poly1305.c]:
```suggestion
        uint32_t *a = &ctx->state[((i) & 3)];
```

File: tests/sys_crypto/tests-crypto-poly1305.c [hunk 2]
```diff
@@ -19,198 +19,299 @@ static const uint8_t msg_1[64] = { 0 };
 static const uint8_t tag_1[16] = { 0 };
 
 static const uint8_t key_2[32] = {
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-    0x36, 0xe5, 0xf6, 0xb5, 0xc5, 0xe0, 0x60, 0x70, 0xf0, 0xef, 0xca, 0x96, 0x22, 0x7a, 0x86, 0x3e
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```
Comment (aabadie) [inline tests/sys_crypto/tests-crypto-poly1305.c]: For all keys I would either leave them as they were before or key 8 values per row. But I would definitely not keep the result of uncrustify.

File: sys/crypto/chacha20poly1305.c (hunk 1 above)
Comment (mtausig) [reply sys/crypto/chacha20poly1305.c]: Reverted the file 69f79847

File: tests/sys_crypto/tests-crypto-poly1305.c (hunk 2 above)
Comment (mtausig) [reply tests/sys_crypto/tests-crypto-poly1305.c]: Reverted the file 69f79847

File: tests/sys_crypto/tests-crypto-chacha20poly1305.c [hunk 3]
```diff
@@ -21,18 +21,27 @@ uint8_t ebuf[1024];
 uint8_t pbuf[1024];
 
 static const uint8_t key_1[32] = {
-    0x80, 0x81, 0x82, 0x83, 0x84, 0x85, 0x86, 0x87, 0x88, 0x89, 0x8a, 0x8b, 0x8c, 0x8d, 0x8e, 0x8f,
-    0x90, 0x91, 0x92, 0x93, 0x94, 0x95, 0x96, 0x97, 0x98, 0x99, 0x9a, 0x9b, 0x9c, 0x9d, 0x9e, 0x9f
+    0x80, 0x81, 0x82, 0x83, 0x84, 0x85, 0x86, 0x87, 0x88, 0x89, 0x8a, 0x8b,
+    0x8c, 0x8d, 0x8e, 0x8f,
```
Comment (aabadie) [inline tests/sys_crypto/tests-crypto-chacha20poly1305.c]: Same issues with keys here. But this time, just revert the changes on keys, not the changes on the whole file :)
Use `git reset -p` or something similar.

File: tests/sys_crypto/tests-crypto-poly1305.c (hunk 2 above)
Comment (aabadie) [reply tests/sys_crypto/tests-crypto-poly1305.c]: There was no need to revert the whole, just the changes I pointed out.

File: tests/sys_crypto/tests-crypto-chacha20poly1305.c (hunk 3 above)
Comment (mtausig) [reply tests/sys_crypto/tests-crypto-chacha20poly1305.c]: Sorry, I'm not gonna do manual code style changes to files I'm not even touching. 
Reverted: 1b5f42a9

Comment (aabadie) [reply tests/sys_crypto/tests-crypto-chacha20poly1305.c]: > I'm not gonna do manual code style changes to files I'm not even touching

Makes sense. Looks good now. Could you just squash your changes into a single commit ?

Comment (mtausig) [reply tests/sys_crypto/tests-crypto-chacha20poly1305.c]: Done.
Btw: Since when *sys/crypto*? The last PRs in this area I've done should always have been prefixed with just *crypto*.

Comment (aabadie) [reply tests/sys_crypto/tests-crypto-chacha20poly1305.c]: > Since when sys/crypto?

Because the changes are applied to files in the sys/crypto module. It would have been even better to put the changes in tests/ in a separate commit but I don't want to bother you much.
